WNPRC_Purchasing/src/org/labkey/wnprc_purchasing/WNPRC_PurchasingController.java

```diff
@@ -90,6 +94,16 @@ public WNPRC_PurchasingController()
         setActionResolver(_actionResolver);
     }
 
+    @NotNull
+    private List<User> getFolderAdmins()
+    {
+        //get only true folder admins, i.e. don't include site admins
+        List<User> adminUsers = SecurityManager.getUsersWithPermissions(getContainer(), Collections.singleton(AdminPermission.class));
+        List<User> siteAdminUsers = SecurityManager.getUsersWithPermissions(getContainer(), Collections.singleton(SiteAdminPermission.class));
```

Comment:
This may not be quite right. It looks like it screens out all site admins, even if they're directly added as folder admins to the purchasing module. We'd want them to be included too, right?

Reply:
Now changed to use role based filtering